### DISCUSSION TITLE
Very nice! About sourcing bellows ->

### DISCUSSION THREAD

**Discussion by nikonowicz** [edgarkech/scalable_field_camera#17] [General]
Just learned about this from someone in my company inbox - I run Standard Cameras. Very cool project! I had a light chuckle about us always being out of stock 😅 sorry y'all. We restock regularly, but folks always snatch up our inventory so quickly. That said, if anyone needs bellows or glass for their builds, feel free to shoot us an email https://standardcameras.com/pages/contact and we can give you the current timeline to next restock. Felt compelled to share just so folks would feel empowered to make their own cameras and make photographs!

**Comment by edgarkech** [comment]
Hi Drew,
causing stock outages for your bellows was never my intention - but your bellows were a perfect fit for my project ;-)
If only you had also a 5x7 version... just joking.
Really appreciate your work with the Standard Camera, although I never built one - I am just more drawn to field cameras.
best regards,
Edgar
